app/assets/javascripts/general.js

```diff
@@ -65,4 +65,54 @@ $(document).ready(function() {
     $this.unbind("mouseup");
     return false;
   });
+
+  $('.js-toggle-delivery-log').on('click', function(e){
```

Comment:
Probably worth changing "Show delivery log" to "Hide delivery log" when its open?